core/src/main/scala/org/zalando/grafter/Visualize.scala

```diff
@@ -0,0 +1,107 @@
+package org.zalando.grafter
+
+import scala.util.matching.Regex
+import org.bitbucket.inkytonik.kiama.relation.{Relation, Tree}
+
+trait Visualize {
+
+  type HashCode = Int
+
+  /**
+   * Generate a representation of the graph induced by the rootComponent
+   * in GraphViz DOT format (http://www.graphviz.org)
+   * A components is taken into account if its package is included and not excluded by
```

Comment:
Thanks, see the fixup where I changed the comment and also the default `filter` function.